<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registerpage</class>
 <widget class="QDialog" name="Registerpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Registerpage{border-image: url(:/new/prefix1/login_image.png);}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Become a New Player!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsrLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PwdLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CheLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Check your Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
